recipes-extended/baseboxd-tools/baseboxd-tools/bundle-debug-info

```diff
@@ -141,12 +141,12 @@ function get_network_state() {
   log_cmd_output ip a
   log_cmd_output ip route list table all
   log_cmd_output ip neigh
-  log_cmd_output bridge -d vlan
   log_cmd_output bridge fdb
```

Comment:
should we maybe move this one as well?

Reply:
This one is a bit weird, this works and produces output even without any bridges, so I think it can stay there.

The fdb also contains any local addresses of interfaces that don't belong to any bridge:

```
ubuntu@bbctrl01:~$ bridge fdb
33:33:00:00:00:01 dev eno1 self permanent
01:00:5e:00:00:01 dev eno1 self permanent
01:80:c2:00:00:00 dev eno1 self permanent
01:80:c2:00:00:03 dev eno1 self permanent
01:80:c2:00:00:0e dev eno1 self permanent
33:33:ff:9c:29:f8 dev eno1 self permanent
33:33:00:00:00:01 dev eno2 self permanent
01:00:5e:00:00:01 dev eno2 self permanent
01:80:c2:00:00:21 dev eno2 self permanent
33:33:ff:23:23:95 dev eno2 self permanent
33:33:00:00:00:01 dev eno3 self permanent
01:00:5e:00:00:01 dev eno3 self permanent
33:33:00:00:00:01 dev eno4 self permanent
33:33:00:00:00:01 dev eno5 self permanent
33:33:00:00:00:01 dev eno6 self permanent
33:33:00:00:00:01 dev eno7 self permanent
33:33:00:00:00:01 dev eno8 self permanent
33:33:00:00:00:01 dev bond1 self permanent
01:80:c2:00:00:21 dev bond1 self permanent
01:00:5e:00:00:01 dev bond1 self permanent
33:33:ff:23:23:95 dev bond1 self permanent
33:33:00:00:00:01 dev bond1.10 self permanent
01:00:5e:00:00:01 dev bond1.10 self permanent
33:33:ff:23:23:95 dev bond1.10 self permanent
```

Reply:
interesting